src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.UnitConverter.UnitTest/UnitHandlerTests.cs

```diff
@@ -78,6 +78,30 @@ public void HandleInvalidModel()
             Assert.AreEqual(0, results.Count());
         }
 
+        [TestMethod]
+        public void HandleOunces()
```

Comment:
What about `to ounce`? Should return two results too. Plese add second test case.

Reply:
Added!